test/boost/sstable_compaction_test.cc

```diff
@@ -5208,6 +5209,7 @@ SEASTAR_TEST_CASE(cleanup_during_offstrategy_incremental_compaction_test) {
             }
             ssts = {}; // releases references
             auto owned_ranges_ptr = make_lw_shared<const dht::token_range_vector>(std::move(owned_token_ranges));
+            t->flush().get();
```

Comment:
This wouldn't be needed if `perform_cleanup_compaction` would get a flush_mode flag defaulted to flush_mode::compacted_tables

Reply:
yup.